packages/vscode-extension/src/webviews/connectionPanel.ts

```diff
@@ -78,6 +78,7 @@ export class ConnectionPanel {
           Uri.file(path.join(extensionPath, 'resources')),
         ],
         enableScripts: true,
+        retainContextWhenHidden: true,
```

Comment:
As mentioned [here](https://code.visualstudio.com/api/extension-guides/webview#persistence) this is not optimal because it leaves the tab running, but it's the most easy to maintain solution. I've tried other solutions and this seems the most easy one to maintain and we can accept the cost of leaving that tab running because when you fill the connection it will always close automatically.